org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -225,6 +225,14 @@ Parameters:
     Type: String
     Default: 'b4e86468-f091-70e2-0ea9-64cf3addcdf1'
 
+  genieProdViewerGroup:      #JC aws-genie-prod-viewers
```

Comment:
no admins or developer role needed for this account?

Reply:
Correct.  We will follow up with a new role that gives app' creators permission to manage secrets (see https://sagebionetworks.jira.com/browse/DCA-35) but not have administrative access.